docs/concepts/nodes.md

```diff
@@ -85,7 +85,7 @@ Limitations of a bootstrap node:
 
 ### Delegate routing node
 
-When IPFS nodes are unable to run Distributed Hash Table (DHT) logic on their own, they _delegate_ the task to a delegate routing node.  Publishing works with arbitrary CID codecs (compression/decompression technology), as the [js-delegate-content module](https://github.com/libp2p/js-libp2p-delegated-content-routing/blob/master/src/index.js#L127-L128) publishes CIDs at the block level rather than the IPLD or DAG level.
+When IPFS nodes are unable to run Distributed Hash Table (DHT) logic on their own, they _delegate_ the task to a delegate routing node.  Publishing works with arbitrary CID codecs (compression/decompression technology), as the [js-delegate-content module](https://github.com/libp2p/js-libp2p-delegated-content-routing/blob/master/src/index.ts) publishes CIDs at the block level rather than the IPLD or DAG level.
```

Comment:
@DannyS03 thought you might have thoughts on where this should link to?

Reply:
> @DannyS03 can you take a look at the links to the LibP2P docs from `concepts/libp2p/`? I was unsure about which pages to link to in a few cases. Basically just wondering if the updated links are the best spots to link to, or if you have other suggestions.
> 
> Also, in `related` (line 9 of the file), there's a link to https://docs.libp2p.io/examples/, which is busted. I looked in the LibP2P docs but couldn't figure out where it should go? Any thoughts?
> 
> The commit diff should have context [398b050](https://github.com/ipfs/ipfs-docs/commit/398b050a733e40dd85bbb57095d0b3abf905afcf)
> 
> Also, can you look at [0faae21#r997293840](https://github.com/ipfs/ipfs-docs/commit/0faae2110206e29a14f2751364338ea47e8060fa#r997293840) and LMK if you have thoughts?

The missing redirect here will also be applied in the restructure PR. We had combined /tutorials/ and /examples/ under /tutorials/ -> https://docs.libp2p.io/tutorials/

Reply:
I believe it would be [this](https://github.com/libp2p/js-libp2p-delegated-content-routing/blob/master/src/index.ts#L122-L123). But if there is a way to avoid hardcoding the reference it would be best as each time the page updates, the lines are subject to change.